examples/bin/register_namespace

```diff
@@ -13,4 +13,17 @@ rescue Temporal::NamespaceAlreadyExistsFailure
   Temporal.logger.info 'Namespace already exists', { namespace: namespace }
 end
 
-
+# Register a variety of search attributes for ease of integration testing
```

Comment:
fixes some tests that were broken in CI, I think based on some recent temporal docker image changes to stop registering these automatically.

Reply:
I think the behavior became inconsistent between how Temporal server is set up in temporalio/auto-setup (which the CI config here uses) and how it is setup in production and the docker-compose. The below code should handle both correctly, but Temporal should restore these to parity. I believe the auto-setup may be using SQL-based enhanced visibility where search attributes are scoped to a namespace and not global like with Elasticsearch.